File: java/test/Ice/facets/FI.java
// **********************************************************************
//
// Copyright (c) 2003-2007 ZeroC, Inc. All rights reserved.
//
// This copy of Ice is licensed to you under the terms described in the
// ICE_LICENSE file included in this distribution.
//
// **********************************************************************

import Test.*;

public final class FI extends _FDisp
{
    public
    FI()
    {
    }

    public String
    callE(Ice.Current current)
    {
        return "E";
    }

    public String
    callF(Ice.Current current)
    {
        return "F";
    }
}

File: cs/test/Ice/facets/FI.cs
// **********************************************************************
//
// Copyright (c) 2003 - 2004
// ZeroC, Inc.
// North Palm Beach, FL, USA
//
// All Rights Reserved.
//
// This copy of Ice is licensed to you under the terms described in the
// ICE_LICENSE file included in this distribution.
//
// **********************************************************************


public sealed class FI : F_Disp
{
    public FI()
    {
    }
    
    public override string callE(Ice.Current current)
    {
        return "E";
    }
    
    public override string callF(Ice.Current current)
    {
        return "F";
    }
}
